scripts/userstyles.yml

```
@@ -300,6 +300,12 @@ userstyles:
       app-link: "https://last.fm"
       current-maintainers: [*anubisnekhet]
       past-maintainers: [*gingeh]
+  lemmy:
+    name: lemmy
```

...for the third time lol, from what I can tell the capitalization of "lemmy" should in fact be "Lemmy", so that needs to be fixed.

```suggestion
    name: Lemmy
```

---

No worries! I should have just made them suggestions in the first place. Also, apologies for being passive aggressive in that comment, that was rude of me!